.travis.yml

```diff
@@ -16,6 +16,10 @@ env:
   - HAZELCAST_IMAGE=hazelcast-oss
 
 install:
+  - |
+    if [ "$HAZELCAST_IMAGE" = "hazelcast-enterprise"]; then
+      docker login registry.redhat.io -u ${RH_REGISTRY_USERNAME} -p ${RH_REGISTRY_PASSWORD}
```

Comment:
One drawback of this unification would be that you need to have access to Red Hat Registry if you want to build the image.

Reply:
I have changed base image from `registry.redhat.io/ubi8/ubi-minimal` to `registry.access.redhat.com/ubi8/ubi-minimal` at the recommendation of @kwart so creds are not required anymore.